GWXtreme/parametrized_eos_sampler.py

```diff
@@ -179,7 +179,61 @@ def save_data(self):
         f.create_dataset('logp',data=np.array(self.logp))
         f.close()
         
+    def parse_samples(self,burn_in_frac = 0.5, thinning = None):
```

Comment:
Minor formatting:
1. put space after `self,`. 
2. keyword arguments should have no space around `=`

Reply:
Done in [2789682](https://github.com/shaonghosh/GWXtreme/pull/52/commits/27896828dbea4db05ac712d9f362fb843239e3b3)